modules/admin_manual/pages/configuration/server/occ_commands/app_commands/richdocuments.adoc

```diff
@@ -0,0 +1,81 @@
+= Collabora Online / Secure View
+
+Marketplace URL: {oc-marketplace-url}/apps/richdocuments[Collabora Online]
+
+Collabora Online allows you to work with all kinds of collabora office documents inside ownCloud for improved productivity. For details please see the
+xref:enterprise/collaboration/collabora_secure_view.adoc[Collabora Online / Secure View] documentation.
+
+`config:app` commands to configure the Collabora Online app.
+
+[source,console]
+----
+ config:app:get|set
+ richdocuments wopi_url            WOPI Server URL
+ richdocuments secure_view_option  Enable Secure View
+ richdocuments watermark_text      Watermark pattern displayed in the document
+ richdocuments secure_view_open_action_default  Open documents in Secure View with watermark by default
+----
+
+== App Configuration
+
+All app configurations are set and queried with the `config:app` command set. The examples below set a value. To query a value use `config:app:get` and the corresponding key without any options or attributes.
+
+== Set the WOPI URL
+
+WOPI Server URL
+
+The following example command sets the WOPI Server to [your-host-public-ip]:8098
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments wopi_url --value='[your-host-public-ip]:8098'
+----
+
+== Set the secure_view_option
+
+Enable Secure View (possible values: true/false, default: false)
+
+The following example command enables or disables secure view globally in the system
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments secure_view_option --value='true'
+----
+
+== Set the watermark_text
+
+Watermark pattern displayed in the document. It can be arbitrarily string. Including keyword {viewer-email} would be replaced with current user in the document session
+
+The following example command sets the watermark pattern displayed in the document.
+
+[source,console,subs="attributes+"]
+----
+{occ-command-example-prefix} config:app:set richdocuments watermark_text --value="Restricted to {viewer-email}"
```

Comment:
```suggestion
Watermark pattern displayed in the document. It can be arbitrarily string. Including keyword \{viewer-email} would be replaced with current user in the document session

The following example command sets the watermark pattern displayed in the document.

[source,console,subs="attributes+"]
----
{occ-command-example-prefix} config:app:set richdocuments watermark_text --value="Restricted to \{viewer-email}"
```
We have to escape curly braces as Antora uses in-text curly braces as variables to be replaced by the build process

Reply:
ok